### PR TITLE
WIP Attach node version when sending stats to /v1/ping_proposal

### PR DESCRIPTION
**Pull request by interro**

### CODE REVIEW

File: server/dto/node_stats_request.go [hunk 1]
```diff
@@ -1,6 +1,7 @@
 package dto
 
 type NodeStatsRequest struct {
```
Comment (zolia) [inline server/dto/node_stats_request.go]: NodeStatsRequest should have comment..

File: server/dto/node_stats_request.go [hunk 2]
```diff
@@ -1,6 +1,7 @@
...
-	NodeKey  string         `json:"node_key"`
-	Sessions []SessionStats `json:"sessions"`
+	NodeKey  	string         `json:"node_key"`
+	Sessions 	[]SessionStats `json:"sessions"`
+	NodeVersion string 			`json:"node_version"`
```
Comment (donce) [inline server/dto/node_stats_request.go]: It seems `go fmt` was not run.

File: server/mysterium_api.go [hunk 3]
```diff
@@ -98,9 +99,12 @@ func (mApi *mysteriumAPI) UnregisterProposal(proposal dto_discovery.ServicePropo
 
 // PingProposal pings service proposal as being alive
 func (mApi *mysteriumAPI) PingProposal(proposal dto_discovery.ServiceProposal, signer identity.Signer) error {
-	req, err := requests.NewSignedPostRequest(mApi.discoveryAPIAddress, "ping_proposal", dto.NodeStatsRequest{
+	statsDto := dto.NodeStatsRequest{
 		NodeKey: proposal.ProviderID,
-	}, signer)
+		NodeVersion: version.GetInfo().BuildNumber,
```
Comment (tadovas) [inline server/mysterium_api.go]: BuildNumber is not NodeVersion. Node version is more semantic version which comes to GetInfo().Branch field (i.e. tag was 0.1.16)

File: server/dto/node_stats_request.go (hunk 1 above)
Comment (interro) [reply server/dto/node_stats_request.go]: added

File: server/dto/node_stats_request.go (hunk 2 above)
Comment (interro) [reply server/dto/node_stats_request.go]: done

File: server/mysterium_api.go (hunk 3 above)
Comment (interro) [reply server/mysterium_api.go]: rewrote NodeVersion in such format: "branch(build number)"

File: server/dto/node_stats_request.go [hunk 4]
```diff
@@ -17,7 +17,9 @@
 
 package dto
 
+// NodeStatsRequest mapped to json structure
 type NodeStatsRequest struct {
-	NodeKey  string         `json:"node_key"`
-	Sessions []SessionStats `json:"sessions"`
+	NodeKey     string         `json:"node_key"`
+	Sessions    []SessionStats `json:"sessions"`
+	NodeVersion string         `json:"node_version"`
```
Comment (Waldz) [inline server/dto/node_stats_request.go]: I suggest add version to dedicated proposal field:
```
ID:          1,
Format:      "service-proposal/0.0.2",
ServiceType: "openvpn",
```

**Pull request closed without merging**